bin/server.py

```diff
@@ -0,0 +1,126 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Server-side of Helpik. """
+
+import logging
+
+import aiohttp
+import bs4
+from aiohttp import web
+from aiohttp.web_request import Request
+
+from bin import config
+
+logging.basicConfig(filename=config.LOG_PATH, level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+
+class Handler(web.View):
+    """Class-based handler of the requests for wiki synopses. """
+
+    def __init__(self, request: Request):
+        super().__init__(request)
+
+        self._params = {
+            'action': 'parse',
+            'prop': 'text',
+            'page': '',
+            'origin': '*',
+            'format': 'json',
+            'formatversion': 2
+        }
+        self._data = ''
+        self._client_language = config.DEFAULT_LANGUAGE
+
+    async def get(self):
+        """Returns content of a page by a page name. """
+
+        synopsis = {}
+
+        if 'pageName' in self.request.query:
+            self._params['page'] = self.request.query['pageName']
+
+        if 'language' in self.request.query:
+            self._client_language = self.request.query['language']
+
+        await self.get_synopsis()
+
+        synopsis['text'] = self._content_parser()
+        synopsis['url'] = config.MEDIAWIKI_API_URL + self._params['page']
+        LOGGER.info('Response to the client`s request')
+
+        if synopsis['text'] == 'None':
+            return web.StreamResponse(status=404)
+
+        return web.json_response(synopsis)
+
+    async def get_synopsis(self):
+        """Parses the content of the wiki page and fetches synopsis. """
+
+        self._params['page'] = f"{self._params['page']}/{self._client_language}"
+        content = await self._request_mediawiki_api()
+
+        if content == '':
+            LOGGER.error('MediaWiki not found')
+            self._data = ''
+            return self._data
+
+        if 'error' in content:
+            self._params['page'] = self._params['page'].replace(f'/{self._client_language}', '/en')
+            content = await self._request_mediawiki_api()
+
+        try:
+            self._data = content
+            self._data = self._data['parse']['text']
+        except (AttributeError, TypeError, KeyError):
+            self._data = ''
+            LOGGER.error('MediaWiki doesn`t return page text')
+
+        return self._data
+
+    async def _request_mediawiki_api(self):
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.get(url=f'{config.MEDIAWIKI_API_URL}',
+                                       params=self._params) as resp:
+                    content = await resp.json()
+                await session.close()
+        except aiohttp.client_exceptions.ClientConnectorError:
+            content = ''
+            LOGGER.error("Can't connect to MediaWiki")
+
+        return content
+
+    def _content_parser(self):
```

Comment:
1) I guess the name is bad. Let's make it simple -- `_strip`.
2) Add a docsting: `Strips HTML tags out of the synopsis.`.